ReaderIOS/ci_scripts/ci_post_clone.sh

```diff
@@ -1,9 +1,9 @@
 #
-//  ci_post_clone.sh
-//  ReaderIOS
-//
-//  Created by Jonas Schiessl on 2/1/25.
-//
+#  ci_post_clone.sh
+#  ReaderIOS
+#
+#  Created by Jonas Schiessl on 2/1/25.
+#
```

Comment:
There’s a typo on line 13 in the key name still. Unless it’s supposed to be that way (cf the Referer header), in which case, toss in a comment to explain. Defaults keys don’t have the instant name recognition of longstanding HTTP headers.

Reply:
Tbh they’re cryptic even spelt properly. Explaining the role each plays in CI and why it’s important to set them would be really helpful.